Stack/Opc.Ua.Core/Security/Certificates/CertificateIdentifier.cs

```diff
@@ -162,18 +162,19 @@ public async Task<X509Certificate2> Find()
         /// <summary>
         /// Loads the private key for the certificate with an optional password.
         /// </summary>
-        public async Task<X509Certificate2> LoadPrivateKey(String password)
+        public async Task<X509Certificate2> LoadPrivateKey(ICertificatePasswordProvider passwordProvider)
```

Comment:
API breaking change here.
I tried to avoid it by keeping both signatures, but that gives a compilation error for ambiguous call at `LoadPrivateKey(null)`

```
        /// <summary>
        /// Loads the private key for the certificate with an optional password.
        /// </summary>
        public async Task<X509Certificate2> LoadPrivateKey(String password)
        {
            return await LoadPrivateKey(new CertificatePasswordProvider(password));
        }
```

Reply:
ok, that's an issue. Maybe I call the new function LoadPrivateKeyEx, so old code will still work?

Reply:
@AlinMoldovean please feel free to merge if you think your concern is fixed...